src/hooks/useReleaseStatus.ts

```diff
@@ -8,18 +8,19 @@ export const useReleaseStatus = (release: ReleaseKind) => {
       return runStatus.Unknown;
     }
 
-    const progressing = release.status.conditions.some((c) => c.reason === 'Progressing');
-    if (progressing) {
-      return runStatus['In Progress'];
-    }
-
-    const succeeded = release.status.conditions.every(
-      (c) => c.reason === 'Succeeded' && c.status === 'True',
+    const succeeded = release.status.conditions.some(
+      (c) =>
+        c.type === ReleaseCondition.Released && c.reason === 'Succeeded' && c.status === 'True',
     );
     if (succeeded) {
       return runStatus.Succeeded;
     }
 
+    const progressing = release.status.conditions.some((c) => c.reason === 'Progressing');
+    if (progressing) {
+      return runStatus['In Progress'];
+    }
+
```

Comment:
We should derive other status from condition type Released only.

Reply:
For more context: Please go through this [thread](https://redhat-internal.slack.com/archives/C038DJAP7HR/p1729019081229139)

Reply:
Specified to the released type now. Thank you for pointing it out.